packages/editor/src/components/blocks/datatablecolumn/DataTableColumn.tsx

```diff
@@ -23,16 +27,35 @@ export const DataTableColumnComponent: ComponentConfig<DataTableColumnProps> = {
   fields: {
     header: { subsection: 'General', label: 'Header', type: 'text' },
     value: { subsection: 'General', label: 'Value', type: 'textBrowser', browsers: ['ATTRIBUTE'], options: { onlyAttributes: 'COLUMN' } },
+    sortable: { subsection: 'General', label: 'Enable Sorting', type: 'checkbox' },
+    filterable: { subsection: 'General', label: 'Enable Filtering', type: 'checkbox' },
     ...baseComponentFields
   },
   quickActions: ['DELETE', 'DUPLICATE']
 };
 
-const UiBlock = ({ header, value }: UiComponentProps<DataTableColumnProps>) => (
+const UiBlock = ({ header, value, sortable, filterable }: UiComponentProps<DataTableColumnProps>) => (
   <Table className='placeholder-table'>
     <TableHeader>
       <TableRow>
-        <TableHead style={{ fontWeight: 'bold' }}>{header}</TableHead>
+        <TableHead style={{ fontWeight: 'bold' }}>
+          <Flex justifyContent='space-between' direction='column' gap={2}>
+            <Flex direction='row' alignItems='center' gap={2}>
+              {header}
+              {sortable && (
+                <div>
+                  <IvyIcon icon={IvyIcons.ArrowRight} rotate={90} />
+                  <IvyIcon icon={IvyIcons.ArrowRight} rotate={270} />
+                </div>
+              )}
+            </Flex>
+            {filterable && (
+              <Flex className='block-search' gap={2} direction='column'>
+                <span className='block-search__input'>Filter By {header}</span>
+              </Flex>
```

Comment:
You are right, flex is not needed anymore 👍 
I think i would not just display an icon as this does not resamble the rendered table and therefore could confuse the low coder, right? 
However one proble with this additional search-input is that now the header does not have an equal height on all columns, which could be fixed by just adding a search icon next to the sort-icon🤔  Otherwise i am not sure if i can somehow easily tell the other columns, that they need to adjust there height 😅 
![grafik](https://github.com/user-attachments/assets/2a3c74f1-aeca-4fda-8dd8-58bd7addcfc9)

Reply:
ok for me with the search input 👍

Reply:
Just saw on this screenshot that the rows per page is empty? maybe you should only show this if you set something? or set it to the default of the paginator?